VALIDATION-SCHEMA.md

```diff
@@ -0,0 +1,116 @@
+# Model Metadata Validation Schema
+The `typeSchema` is an optional part of the model_metadata.yaml file that consists of `input_requirements` and 
+`output_requirements` fields.  Both requirements fields are optional, allowing the user to specify on
+input or output requirements, or both as is appropriate for their model.  
+
```

Comment:
this reads pretty awkwardly. seems redundant with above. pls reword

Reply:
maybe define what a "requirement might be, like `allowing the user to specify exactly what kinds of data their task expects`